### DISCUSSION TITLE
Tridents can be thrown but not snowballs

### DISCUSSION THREAD

**Discussion by iiGalixy** [Mojang/minecraft-editor#25] [Bugs]
Tridents can be thrown but not snowballs, arrows, xp bottles, fishing hook

how to recreate 
throw trident, then throw a snowball

**Comment by ihategravel2** [comment]
Things such as snowballs, arrows, XP bottles, and fishing hooks are technically being thrown, but they only show up after being unpaused.

**Comment by minecraftchloe** [comment]
In a paused simulation, what would be your expected interaction? After just testing it, it appears to be working as intended/designed on our side, but would love to know what you expected.
